templates/assets/css/vlabs-style.css

```diff
@@ -1,180 +1,303 @@
+  overflow-x: hidden;
 }
 
 .vlabs-header {
-    border-bottom: 8px solid #ff6600;
-    font-family: 'Raleway', sans-serif;
+  border-bottom: 8px solid #ff6600;
+  font-family: "Raleway", sans-serif;
 }
 
 .vlabs-page-main {
-    font-size: 1rem;
-    font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  font-family: "Open Sans", sans-serif;
 }
 
 .vlabs-lab-name {
-    font-size: 1.7rem;
+  font-size: 1.7rem;
 }
 
 .vlabs-footer {
-    font-size: 14px;
-    background: rgb(17, 17, 17);
-    font-family: 'Raleway', sans-serif;
+  font-size: 14px;
+  background: rgb(17, 17, 17);
+  font-family: "Raleway", sans-serif;
 }
 
 .vlabs-footer-sect-name {
-    border-width: 0.15rem;
-    border-style: solid;
-    border-image: linear-gradient( to right, #3EC1D5 20%, #555 0% ) 0% 0% 100% 0%;
+  border-width: 0.15rem;
+  border-style: solid;
+  border-image: linear-gradient(to right, #3ec1d5 20%, #555 0%) 0% 0% 100% 0%;
 }
 
 .vlabs-lab-name {
-    color: #2C99CE;
+  color: #2c99ce;
 }
 
 .vlabs-page-content {
-    font-size: 1.2rem;
-    overflow-y: hidden;
-    font-family: 'Raleway', sans-serif;
+  font-size: 1.2rem;
+  overflow-y: hidden;
+  font-family: "Raleway", sans-serif;
 }
 
 .social-links > a {
-    color: #fff;
-    border-radius: 50%;
-    width: 36px;
-    height: 36px;
+  color: #fff;
+  border-radius: 50%;
+  width: 36px;
+  height: 36px;
 }
 
 .sidebar a {
-    color: #3e6389;
+  color: #3e6389;
 }
 
-
 .sidebar .tasks a {
-    color: #5c5c5c;
+  color: #5c5c5c;
 }
 
 .sidebar a.current-item {
-    color: #ff6600;
+  color: #ff6600;
 }
 
 .sidebar .tasks {
-    font-size: 1rem;
+  font-size: 1rem;
 }
 
 .sidebar {
-    border-right: 2px dotted #89a7c4;
-    font-size: 1.2rem;
-    font-weight: bold;
-    overflow-x: hidden;
-    box-sizing: content-box;
+  border-right: 2px dotted #89a7c4;
+  font-size: 1.2rem;
+  font-weight: bold;
+  overflow-x: hidden;
+  box-sizing: content-box;
 }
 
 .vlabs-header a {
-    margin: 0 1rem;
-    padding: 0.5rem 1rem;
-    color: #2C98CD;
+  margin: 0 1rem;
+  padding: 0.5rem 1rem;
+  color: #2c98cd;
 }
 
-.vlabs-header #headerNavbar a:hover {    
-    background: #77BB41;
-    color: #fff;
-    border-radius: 10px;
+.vlabs-header #headerNavbar a:hover {
+  background: #77bb41;
+  color: #fff;
+  border-radius: 10px;
 }
 
 .vlabs-header #headerNavbar a {
-    border-radius: 10px;
-    transition: 0.3s;
+  border-radius: 10px;
+  transition: 0.3s;
 }
 
-.breadcrumbs, .breadcrumbs a, .breadcrumbs span {
-    font-size: 1.6rem;
-    color: #337ab7;
+.breadcrumbs,
+.breadcrumbs a,
+.breadcrumbs span {
+  font-size: 1.6rem;
+  color: #337ab7;
 }
 
 .page-name {
-    color: #337ab7;
+  color: #337ab7;
 }
 
 /*pre-test and post-test page styling fix*/
 
 .answers {
-    font-size: 1rem;
-    display: flex;
-    flex-direction: column;
-    margin-bottom: 1rem;
+  font-size: 1rem;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1rem;
+}
+
+.answers.grid {
+  display: grid;
```

Comment:
Why are we using grid over the responsive styling of Bootstrap?

Reply:
This was redundant styling css sir, even without it was working, removed it with an update.